Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -60,23 +54,32 @@ private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
             _hoverWrapper = hoverWrapper;
         }
 
-        private void Gripper_KeyDown(object sender, KeyRoutedEventArgs e)
+        private void CreateGripperDisplay()
         {
-            var gripper = sender as GridSplitterGripper;
-
-            if (gripper == null)
+            if (_gripperDisplay == null)
             {
-                return;
+                var gripperText = _resizeDirection == GridResizeDirection.Columns ? GripperBarVertical : GripperBarHorizontal;
+                _gripperDisplay = new TextBlock
+                {
+                    FontFamily = new FontFamily(GripperDisplayFont),
+                    HorizontalAlignment = HorizontalAlignment.Center,
+                    VerticalAlignment = VerticalAlignment.Center,
+                    Foreground = GripperForeground,
+                    Text = gripperText
+                };
             }
+        }
 
+        private void GridSplitter_KeyDown(object sender, KeyRoutedEventArgs e)
```

Comment:
What about override on key down instead of registering new event ?
`
        protected override void OnKeyDown(KeyRoutedEventArgs e)
        {
            var step = 1;
            var ctrl = Window.Current.CoreWindow.GetKeyState(VirtualKey.Control);
            if (ctrl.HasFlag(CoreVirtualKeyStates.Down))
            {
                step = 5;
            }

            if (_resizeDirection == GridResizeDirection.Columns)
            {
                if (e.Key == VirtualKey.Left)
                {
                    HorizontalMove(-step);
                }
                else if (e.Key == VirtualKey.Right)
                {
                    HorizontalMove(step);
                }
                else
                {
                    return;
                }

                e.Handled = true;
                return;
            }

            if (_resizeDirection == GridResizeDirection.Rows)
            {
                if (e.Key == VirtualKey.Up)
                {
                    VerticalMove(-step);
                }
                else if (e.Key == VirtualKey.Down)
                {
                    VerticalMove(step);
                }
                else
                {
                    return;
                }

                e.Handled = true;
            }

            base.OnKeyDown(e);
        }
`